guide/interactions/registering-slash-commands.md

```diff
@@ -161,3 +161,62 @@ const data = {
 	}],
 };
 ```
+
+## Advanced deployment
+
+Advanced users should make use of a deploy script to deploy their commands when needed. In this section we'll be using a script that is usable in conjunction with the [slash command handler](/command-handling) from the command handling section.
+
+First off, install the discord.js REST module by running `npm i @discordjs/rest` in your terminal.
```

Comment:
Maybe put a warning stating the module is in canary/pre-alpha (as the repo states), because there might be bugs?

Reply:
Considering the module has high test coverage, this is redundant and not needed. Esp not with the usage portrait here.

Reply:
Right, that's true. But after all, it was just a suggestion. 🤷🏼‍♂️ 
I appreciate the opinion!